ksqldb-common/src/main/java/io/confluent/ksql/util/ConsistencyOffsetVector.java

```diff
@@ -271,4 +205,35 @@ public static boolean isConsistencyVectorEnabled(final Map<String, Object> reque
 
     return KsqlConfig.KSQL_QUERY_PULL_CONSISTENCY_OFFSET_VECTOR_ENABLED_DEFAULT;
   }
+
+  @Override
+  public boolean lessThanOrEqualTo(final OffsetVector other) {
+    throw new UnsupportedOperationException("Unsupported");
+  }
+
+  @JsonIgnore
+  @Override
+  public List<Long> getDenseRepresentation() {
+    throw new UnsupportedOperationException("Unsupported");
+  }
+
+  @JsonIgnore
+  @Override
+  public boolean dominates(final OffsetVector other) {
+    throw new UnsupportedOperationException("Unsupported");
```

Comment:
I guess that's an artifact of the OffsetVector interface we are implementing. Are we using OffsetVector anywhere?

Reply:
The interface per se, no we are not using it currently anywhere. But both the `PushOffsetVector` and the `ConsistencyOffsetVector` are implementing it

Reply:
Yeah, I think a good further cleanup would be to drop that interface. It seems to be causing more harm than good, and it also doesn't seem like we have discovered any desire to actually unify the continuation tokens of push queries with the consistency tokens of pull queries.

Since this is a private API, though, it seems better to just merge this PR and have another follow-up to drop the interface.